src/components/figma-comparer/FigmaComparer.ts

```diff
@@ -15,13 +16,13 @@ import css from "./figma-comparer.scss";
 
 export class FigmaComparer implements Component {
     private readonly STEPS: string[] = [
-        "Figma Login",
-        "Screen Selection",
-        "Tasks Edition",
+        i18next.t('figma:comparer.steps.login'),
+        i18next.t('figma:comparer.steps.screenSelection'),
+        i18next.t('figma:comparer.steps.tasksEdition'),
     ];
 
     private readonly DEFAULT_CANCEL_BUTTON_CONFIG: IButtonConfig = {
-        text: "Cancel",
+        text: i18next.t('figma:comparer.buttonText.cancel'),
```

Comment:
If there was a cancel button outside of the compare figma, it wouldn't look nice if the name was `figma:comparer`

So I think we need to separate some common texts like cancel, ok, previous,...

Reply:
I see that the `DEFAULT_CANCEL_BUTTON_CONFIG` just use inside FigmaComparer class. It is not exported to use outside the class so I think it will not confuse if we define i18n like this.